build/networking/README_DOCKER.md

```diff
@@ -77,226 +75,122 @@ container up and running, and should see the container prompt like below:
 
 `c5efb1a949ad` is your container id as displayed in command `docker ps -a`
 
-As your P4-OvS should be up and running, you could see the ovsdb-server
-and ovs-vswitchd process with the `ps -ef | grep ovs` command.
-
-## Section 2: Running example setup ( VM1 <-> IPDK Container <-> VM2 )
-Below commands will help you setup traffic between 2 VMs on your host with the
-IPDK container as a P4 program enabled vswitch switching traffic between them.
-
-Pre-requisite:
-- Container in Section 1 should be up and with OvS running
-- If you are running in a VM then make sure you have nested virtualization
-  enabled on your guest VM
-- QEMU and KVM should be installed and running
+## Run Networking Recipe
+Below commands will help you setup the environment for Networking recipe and
+start the infrap4d
 
-### Example description
-
-The demo environment is easily setup. The command below will set the environment up and allow for simple testing using the P4OVS container:
+### Set up the environment required by infrap4d
 
 ```
-ipdk demo
+export IPDK_RECIPE=/root/networking-recipe
+export DEPEND_INSTALL=/root/networking-recipe/deps_install
+export SDE_INSTALL=/root/p4-sde/install
+export LD_LIBRARY_PATH=$IPDK_RECIPE/install/lib/:$IPDK_RECIPE/install/lib64/:$SDE_INSTALL/lib:$SDE_INSTALL/lib64:$DEPEND_INSTALL/lib:$DEPEND_INSTALL/lib64:$SDE_INSTALL/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
```

Comment:
Isnt there also a script that sets up the path?

The documentation from this page: https://github.com/ipdk-io/networking-recipe/blob/main/docs/ipdk-dpdk.md

has the following line
`source ./scripts/dpdk/setup_env.sh $IPDK_RECIPE $SDE_INSTALL $DEPEND_INSTALL`